### PR TITLE
I have just added my repo file

### PR DESCRIPTION
**Pull request by Piyush01-672**

### CODE REVIEW

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]
change the name of the file
follow
name - project title - date

**Comment by Piyush01-672** [comment]
okay!

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]
remove this file 
image

**Comment by Piyush01-672** [comment]
Hlo! I have just changed the name of file as you said. Thank you for the guidence 👍

File: Submit/medichain by piyush and simar.png
Comment (AKSHITHA-CHILUKA) [inline Submit/medichain by piyush and simar.png:1]: image
remove this file

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]

**Pull request merged**